protocol/x/prices/types/message_update_market_param.go

```diff
@@ -13,8 +14,15 @@ func (msg *MsgUpdateMarketParam) GetSigners() []sdk.AccAddress {
 }
 
 func (msg *MsgUpdateMarketParam) ValidateBasic() error {
-	if msg.Authority == "" {
-		return errorsmod.Wrapf(ErrInvalidAuthority, "authority cannot be empty")
+	if _, err := sdk.AccAddressFromBech32(msg.Authority); err != nil {
```

Comment:
I see that we are not using this pattern in all the module tx msgs. Examples:
* [blocktime.MsgUpdateDowntimeParms](https://github.com/dydxprotocol/v4-chain/blob/46011c0871519940d8c05672d3dd8701435b1178/protocol/x/blocktime/types/tx.go#L12)
* bridge
* clob (all the update ones)
* etc

Can you or Tian work on updating these to be consistent? I think it might make sense to change all of them in a single PR